src/internal/m365/collection/drive/item_collector_test.go

```diff
@@ -38,15 +38,6 @@ const (
 	userMysiteNotFound    = "ResourceNotFound User's mysite not found"
 )
 
-func odErr(code string) *odataerrors.ODataError {
```

Comment:
nice! Quick heads up, we also duplicate this func in services/*/enabled_test.go . See [example](https://github.com/alcionai/corso/blob/1cbabdb73a14854fc97fad87f835ab1edac5e7b7/src/internal/m365/service/exchange/enabled_test.go#L52).

These can be cleaned up as well.

Reply:
thanks for the heads up!